front-end/src/pages/EditEventDetails.jsx

```diff
@@ -1,103 +1,58 @@
-import { useNavigate, useParams } from "react-router-dom";
-import { useEffect, useState } from "react";
-import { Container, Row, Col, Button, Form } from "react-bootstrap";
-import { getInterestCategories } from "../utilities/InterestCategoriesUtilities";
-import { deleteEvent, getEventDetails, updateEventDetails } from "../utilities/EventUtilities";
-import LocationSearchMap from "../components/LocationSearchMap";
+import React, { useEffect, useState } from 'react';
```

Comment:
Again with React here - is there a purpose? Just curious

Reply:
noope lol